services/gateway/app.js

```diff
@@ -129,13 +131,16 @@ broker.createService({
 	},
 	methods,
 	events: {
-		'block.change': (payload) => sendSocketIoEvent('update.block', payload),
-		'round.change': (payload) => sendSocketIoEvent('update.round', payload),
-		'forgers.change': (payload) => sendSocketIoEvent('update.forgers', mapper(payload, {
-			data: ['data', delegateResponse],
-			meta: {},
+		'block.change': (payload) => sendSocketIoEvent('update.block', mapper(payload, {
+			data: ['data', blockResponse],
+			meta: metaResponse,
+		})),
+		'transactions.confirmed': (payload) => sendSocketIoEvent('update.transactions.confirmed', mapper(payload, {
+			data: ['data', transactionResponse],
+			meta: metaResponse,
 		})),
-		'transactions.confirmed': (payload) => sendSocketIoEvent('update.transactions.confirmed', payload),
+		'round.change': (payload) => sendSocketIoEvent('update.round', payload),
+		'forgers.change': (payload) => sendSocketIoEvent('update.forgers', payload),
 		'update.fee_estimates': (payload) => sendSocketIoEvent('update.fee_estimates', payload),
```

Comment:
Taking the chance we're here, could you also take care of those mappings please?

Reply:
Sure, I will do that.